v2.19.9 release (#4924)
circleci/circleci-docs: jekyll/_cci2/v.2.19-overview.adoc

* Classic Load Balancer is no longer available from v2.19.02 due to the ciphersuite changes listed in the notes below. CircleCI no longer accepts requests from https://docs.aws.amazon.com/elasticloadbalancing/latest/classic/elb-ssl-security-policy.html[Classic Load Balancer], so you should move to https://docs.aws.amazon.com/elasticloadbalancing/latest/network/introduction.html[Network Load Balancer (NLB)] or https://docs.aws.amazon.com/elasticloadbalancing/latest/application/introduction.html[Application Load Balancer (ALB)].

## What's New in v2.19.09

> * Better handling of non-alphanumeric characters in authentication passwords for Docker executors. Fixed a bug that was preventing the use of `~` and `?` in the <<private-images#docker-executor,Docker ID password field under the `auth` key>>.

Author: ```suggestion
* Better handling of non-alphanumeric characters in authentication passwords for Docker executors. Fixed a bug that was preventing the use of `~` and `?` in the https://circleci.com/docs/2.0/private-images/#docker-executor[Docker ID password field under the `auth` key].
```

> * Better handling of non-alphanumeric characters in authentication passwords for Docker executors. Fixed a bug that was preventing the use of `~` and `?` in the [Docker ID password field under the `auth` key]({{ '/2.0/private-images/#docker-executor' | docs_url }}).

Author: ```suggestion
* Better handling of non-alphanumeric characters in authentication passwords for Docker executors. Fixed a bug that was preventing the use of `~` and `?` in the <<private-images#docker-executor,Docker ID password field under the `auth` key>>.
```

* Classic Load Balancer is no longer available from v2.19.02 due to the ciphersuite changes listed in the notes below. CircleCI no longer accepts requests from https://docs.aws.amazon.com/elasticloadbalancing/latest/classic/elb-ssl-security-policy.html[Classic Load Balancer], so you should move to https://docs.aws.amazon.com/elasticloadbalancing/latest/network/introduction.html[Network Load Balancer (NLB)] or https://docs.aws.amazon.com/elasticloadbalancing/latest/application/introduction.html[Application Load Balancer (ALB)].

## What's New in v2.19.09

* Better handling of non-alphanumeric characters in authentication passwords for Docker executors. Fixed a bug that was preventing the use of `~` and `?` in the [Docker ID password field under the `auth` key]({{ '/2.0/private-images/#docker-executor' | docs_url }}).
* Improved the way commit statuses are handled to reduce instances of them being published out of order in certain situations.

> * Security vulnerability patch for the `VM-scaler` service.

Author: ```suggestion
* General security updates.
```
